pkg/inject/inject.go

```diff
@@ -869,3 +920,8 @@ func sortedKeys(m map[string]string) []string {
 
 	return keys
 }
+
+//IsNamespace checks if a given config is of type namespace
+func (conf *ResourceConfig) IsNamespace() bool {
```

Comment:
I dont think we need this anymore. We can use the `IsWorkloadNamespace()` everywhere as both of these functions are doing the same.